doc/guides/tfm/testsuites.rst

```diff
@@ -0,0 +1,51 @@
+Test Suites
+###########
+
+TF-M includes two sets of test suites:
+
+* tf-m-tests - Standard regression tests, to verify the NS interface
+* psa-arch-tests - Test suites for specific APIs (secure storage, etc.)
+
+These test suites can be run from Zephyr via an appropriate sample application
+in the samples/tfm_integration folder.
+
+PSA Regression Tests
+********************
+
+The regression test suite can be run via ``tfm_regression_test``, for
+example:
+
+   .. code-block:: bash
+
+     $ west build samples/tfm_integration/tfm_regression_test/ \
+       -p -b mps2_an521_ns -t run
+
+This sample tests the NS/S integration via the PSA APIs, amongst other things,
```

Comment:
Have you swapped the descriptions of the tests? This sounds more like the PSA Arch Tests to me, and the description for PSA Arch Tests sounds more like the TF-M Regression tests

Reply:
There is some overrlap, but for the purpose of PSA certification the regression test results are what you have to send to demonstrate proper S/NS integratiton via the PSA APIs. The Arch tests (at last time I did the certification, it may have changed) were informative but not required.

Reply:
Sorry, looking again this has changed since the 1.0 certification! I'll update the documentation accordingly. Thanks for the heads up.

Reply:
@andvib Do you mind having a new look?

Reply:
Looks good!